.github/workflows/main_test_itensors_base_macos_windows.yml

```
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         version:
-          - '1.6'
+          - '1.10'
```

```suggestion
          - 'lts'
```
This will automatically expand to 1.10, or whatever the current LTS is. Same thing elsewhere.